assets/scss/style.scss

```diff
@@ -143,3 +143,9 @@ a {
     transition: all 0.2s;
     opacity: 0;
 }
+
+@media (width >= 1400px) {
```

Comment:
why is this needed?

Reply:
Bootstrap 5 has an extra media query. So I'm limiting how much the card can expand